gravatar/build.gradle.kts

```diff
@@ -11,9 +11,7 @@ plugins {
     alias(libs.plugins.dokka)
 }
 
-val sdkVersion = providers.exec {
-    commandLine("git", "describe", "--tags", "--abbrev=0")
-}.standardOutput.asText.get().trim()
+val sdkVersion: String by rootProject.extra
```

Comment:
It's not a problem with this PR because we did the same before, but we are sending the previous version while developing the next iteration. It's not a big deal but wdyt about creating a ticket with low priority to improve this?

Reply:
Hmmm would it be ok if the version during progress be sth like `2.1.0-{hash}`? Where 2.1.0 is the current version? That should be simple and likely only require removing the `--abbrev=0` part.

Reply:
👍 Sounds good to me.

Reply:
Done